### PR TITLE
[doc] admin docs

### PR DESCRIPTION
**Pull request by varortz**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

## :tanabata_tree: Description

Adds documentation for IJP admin. 

### :palm_tree: What's new in this PR
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- adds the admin intro
- adds admin docs for how to use retool
- adds admin docs about LegalServer
- adds admin docs about Brevo

### :evergreen_tree: Screenshots
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @kyrenetam at the bottom."

## :deciduous_tree: How to review
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## :seedling: Next steps 
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- Replace the screenshots on the Retool Admin Dashboard docs with screenshots where it is less obvious that we're using dummy data.

### :information_source: Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### :potted_plant: Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @varortz

### CODE REVIEW

File: docs/admin/retool.md [hunk 1]
```diff
@@ -3,3 +3,159 @@ icon: wrench
...
+
+::: info Note:
+Case-Specific listings cannot be edited through the Retool Admin Dashboard. Case-Specific listings should be updated on LegalServer.
+Refer to this page for more information regarding LegalServer and Case-Specific listings.
+<!-- Link to the Retool developer documentation! -->
+:::
+
+Similarly, to edit a listing, you will need to access the form in the tabbed container. Clicking on the "Edit" tab will allow you to access the form to edit an existing listing. Once all edits have been made, clicking on the "Update Listing" button at the bottom of the form should upload the listing to the database, making all edits visible to the volunteers.
+
+![creating a limited case assignment listing](/assets/image/retool-edit-listing.png)
```
Comment (varortz) [inline docs/admin/retool.md]: should be "editing" not "creating"

File: docs/admin/retool.md [hunk 2]
```diff
@@ -2,4 +2,157 @@
 icon: wrench
 ---
 
-# Retool
+# Retool Dashboard
```
Comment (varortz) [inline docs/admin/retool.md]: change to Admin Dashboard

**Review by varortz — COMMENTED** [review]
Wondering if I could get some feedback on whether we should split up the Retool Admin Dashboard Sections into multiple sub-pages.

**Comment by jinkang-0** [comment]
> Wondering if I could get some feedback on whether we should split up the Retool Admin Dashboard Sections into multiple sub-pages.

if it makes sense for you! I think it could be easier for admins to navigate if we have a separate page for each dashboard section

**Pull request merged**